fuse_graphs/src/hash_graph.cpp

```diff
@@ -426,21 +427,26 @@ ceres::Solver::Summary HashGraph::optimize(const ceres::Solver::Options& options
 }
 
 ceres::Solver::Summary HashGraph::optimizeFor(
-  const ros::Duration& max_optimization_time,
+  const rclcpp::Duration& max_optimization_time,
   const ceres::Solver::Options& options)
 {
-  auto start = ros::Time::now();
+  auto clock = rclcpp::Clock::Clock(RCL_STEADY_TIME);
```

Comment:
I'd recommend passing a clock so that a caller may use a ROS_TIME clock if they want.